com.reprezen.swagedit.openapi3/resources/templates.xml

```diff
@@ -0,0 +1,328 @@
+
+<!--
+	LINKS 
+ -->
+
+ 	<template name="response link" 
+       id="com.reprezen.swagedit.openapi3.templates.response_link" 
+       description="server response link template" 
+       context="com.reprezen.swagedit.openapi3.templates.response" 
+       enabled="true">links:
+            ${name}:
+              $$ref: ${ref}</template>
+
+<!--
+	PARAMETERS 
+ -->
+ 
+ <template name="path parameter" 
+       id="com.reprezen.swagedit.openapi3.templates.path_parameter_object" 
+       description="path parameter template" 
+       context="com.reprezen.swagedit.openapi3.templates.parameter_object" 
+       enabled="true">name: ${name}
+in: path
+description: ${ID}
+required: true
+type: ${integer}
+format: int64
+</template>
+
+     <template name="header parameter" 
+       id="com.reprezen.swagedit.openapi3.templates.header_parameter_object" 
+       description="header parameter template" 
+       context="com.reprezen.swagedit.openapi3.templates.parameter_object" 
+       enabled="true">name: ${name}
+in: header
+description: ${ID}
+required: true
+type: ${integer}
+format: int64
+</template>
+
+     <template name="query parameter" 
+       id="com.reprezen.swagedit.openapi3.templates.query_parameter_object" 
+       description="query parameter template" 
+       context="com.reprezen.swagedit.openapi3.templates.parameter_object" 
+       enabled="true">name: ${name}
+in: query
+description: ${ID}
+required: true
+type: ${integer}
+format: int64
+</template>
+
+     <template name="form data parameter" 
+       id="com.reprezen.swagedit.openapi3.templates.formdata_parameter_object" 
+       description="form data parameter template" 
+       context="com.reprezen.swagedit.openapi3.templates.parameter_object" 
+       enabled="true">name: ${name}
+in: formData
+description: ${ID}
+required: true
+type: ${integer}
+format: int64
+</template>
+
+     <template name="body parameter (request)" 
+       id="com.reprezen.swagedit.openapi3.templates.body_parameter_object" 
+       description="body parameter (request) template" 
+       context="com.reprezen.swagedit.openapi3.templates.parameter_object" 
+       enabled="true">name: ${name}
+in: body
+description: ${ID}
+required: true
+schema:
+  $$ref: ${model}
+</template>
+ 
+<!--
+	SCHEMA 
+ -->
+ 
+	<template name="schema" 
+       id="com.reprezen.swagedit.openapi3.templates.schema" 
+       description="Schema definition template" 
+       context="com.reprezen.swagedit.openapi3.templates.components.schemas" 
+       enabled="true">${type}:
+      required:
+        - ${property}
+      properties:
+        ${property}:
+          type: integer
+          format: int64</template>
+
+<!--
+	CALLBACKS 
+ -->
+ 
+ 	<template name="callbacks" 
+       id="com.reprezen.swagedit.openapi3.templates.callbacks" 
+       description="Callback definition template" 
+       context="com.reprezen.swagedit.openapi3.templates.components" 
+       enabled="true">callbacks:
+      ${name}:
```

Comment:
There are several white spaces in this code template:
<img width="482" alt="screen shot 2017-05-16 at 4 34 31 pm" src="https://cloud.githubusercontent.com/assets/644582/26126966/a363a936-3a55-11e7-8f56-e73dd2425851.png">

 and YEdit does not like it(shows and error)